File: Semester 2/Research Project/2020_AON01/Stage_API/Stage_API.Domain/Classes/Comment.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace Stage_API.Domain.Classes
{
    public class Comment
    {
        public int Id { get; set; }
        public User User { get; set; }
        public int UserId { get; set; }
        public Stagevoorstel Stagevoorstel { get; set; }
        [Required]
        public int StagevoorstelId { get; set; }
        public DateTime Date { get; set; }
        [Required]
        public string Text { get; set; }
    }
}

File: Semester 2/Research Project/2020_AON01/Stage-Frontend/src/app/classes/comment.ts
import { User } from '@app/classes/user';
import { Stagevoorstel } from '@app/classes/stagevoorstel';

export class Comment {
  user: User;
  userId: number;
  stagevoorstel: Stagevoorstel;
  stagevoorstelId: number;
  date: string;
  text: string;
}
